projects/packages/my-jetpack/_inc/components/stats-section/cards.jsx

```diff
@@ -1,10 +1,22 @@
-import { __ } from '@wordpress/i18n';
+import { sprintf, __, _n } from '@wordpress/i18n';
 import { Icon, commentContent, people, starEmpty } from '@wordpress/icons';
 import React from 'react';
+import formatNumber from '../../utils/format-number';
 import CountComparisonCard from './count-comparison-card';
+import createStatDiffText from './create-stat-diff-text';
 import eye from './eye';
 import styles from './style.module.scss';
 
+const createStatSRText = ( countStat, count, previousCount ) => {
```

Comment:
To nitpick, I think JSDoc documentation for this helper function would make sense here.